app/presenters/project_presenter.rb

```diff
@@ -29,6 +29,10 @@ def git_remote
     stage&.git_remote
   end
 
+  def auto_deploys?
+    @project.stages.any? { |s| s.deploy_strategies.any? }
```

Comment:
A `DeployStrategy` might exist that isn't `automatic`. All of ours currently have `automatic: true`, but I defined that field thinking that this app might want to know _how_ to deploy some projects even if it didn't happen automatically (so, e.g., it could offer a deploy button).

For consistency, I might make this method `deploy_strategies?` and keep the logic as it is now, or keep the "auto" language but update the logic to do `s.deploy_strategies.any?(&:automatic?)`.